### ISSUE TITLE
Uncaught TypeError: simpleParallax is not a constructor

### ISSUE THREAD

**Issue by mikeingz** [geosigno/simpleParallax#17]
Hello,
When I try to use the last version (5.0.0), I get this error message:

`Uncaught TypeError: simpleParallax is not a constructor`

But when I use an older version (4.2.1), everything works as expected.

Is there something special to do to make the last version work?

Thanks

**Comment by geosigno** [comment]
Hi mikeingz,
No you shouldn't have anything special to do to make that new version works.
How do you use simpleParallax? via npm? or directly in a script tag?
Can you provide some material so I can help.
Thanks.

**Comment by mikeingz** [comment]
I am using in a script tag, with Wordpress.
Here is a temporary page : http://auburn-guanaco.w6.wpsandbox.pro/parallax/

The script is loaded (in the footer), and I have just added an image, with the same example as in your page:

```
var images = document.querySelectorAll('img');
new simpleParallax(images);
```

**Comment by Juniorr452** [comment]
Same issue here!

**Comment by geosigno** [comment]
This should be fixed with [5.0.1](https://github.com/geosigno/simpleParallax/releases/tag/5.0.1).
Thanks to have spotting that.
